app/models/profile.rb

```diff
@@ -69,23 +81,13 @@ def deactivate(reason)
   end
 
   def deactivate_for_fraud_review
-    update!(
-      active: false,
-      fraud_review_pending: true,
-      fraud_rejection: false,
-      fraud_review_pending_at: Time.zone.now,
-      fraud_rejection_at: nil,
-    )
+    fraud_review
+    update!(active: false)
```

Comment:
Should `active: false` be applied automatically in response to the transition from `fraud_none` -> `fraud_review`?

Reply:
Side note, and unrelated to this PR: should this method be `deactivate_for_fraud_review!` since it is writing to to the db?

Reply:
I'll catalog all the calls and see if any shouldn't.

Reply:
The following methods modify the `active` attribute:
`Profile#activate`,
`Profile#deactivate(reason)`,
`Profile#deactivate_for_fraud_review`,
`Profile#reject_for_fraud(notify_user:)`
I would recommend a future PR to refactor the `active` behavior.

Reply:
`!` doesn't exclusively indicate a database change, but those same 4 methods all call `update!`, so I think you're right, and they should all have a `!`.